GroupRepresentationsForCAP/examples/SemisimpleCategoryTest.g

```diff
@@ -18,22 +18,30 @@ chi_3 := GIrreducibleObject( irr[3] );
 
 semisimple_cat := SemisimpleCategory( Q, membership_function, chi_3, "S3Ass.g", true );
 
+Qmat := UnderlyingCategoryForSemisimpleCategory( semisimple_cat );
+
 a := SemisimpleCategoryObject( [ [ 1, chi_1 ], [ 2, chi_2 ], [ 3, chi_3 ] ], semisimple_cat );
 
 b := SemisimpleCategoryObject( [ [ 1, chi_2 ], [ 2, chi_3 ] ], semisimple_cat );
 
-alpha := VectorSpaceMorphism( VectorSpaceObject( 2, Q ), HomalgMatrix( [ [ 1 ], [ -1 ] ], 2, 1, Q ), VectorSpaceObject( 1, Q ) );
+alpha := VectorSpaceMorphism( MatrixCategoryObject( Qmat, 2 ), HomalgMatrix( [ [ 1 ], [ -1 ] ], 2, 1, Q ), MatrixCategoryObject( Qmat, 1 ) );
```

Comment:
Could you please define all three objects beforehand?

Reply:
Done. I cleaned it up in general a bit more.